.travis.yml

```diff
@@ -1,12 +1,14 @@
 # Test
-sudo: required
```

Comment:
Why?

Reply:
[Travis are now recommending removing the __sudo__ tag](https://blog.travis-ci.com/2018-11-19-required-linux-infrastructure-migration).  Mention  of the __sudo:__ tag has been removed from all Travis docs because the sudo command is now _always_ available and there is no way to turn it off.

Reply:
Ok, that should be a separate commit. It's unrelated to linting python.